<launch>

  <node pkg="yesense_imu" type="yesense_imu_node" name="yesense_imu_node" required="true" output = "screen" >
    <param name="yesense_port"      type="string" value="/dev/ttyUSB0"/>
    <param name="yesense_baudrate"  type="int"    value="460800"/>
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find yesense_imu)/rviz/yesense.rviz" required="true"/>

</launch>
